helpers/hdocker.py

```diff
@@ -729,6 +729,8 @@ def convert_pandoc_cmd_to_arguments(cmd: str) -> Dict[str, Any]:
     # Parse known arguments and capture the rest.
     args, unknown_args = parser.parse_known_args(cmd)
     _LOG.debug(hprint.to_str("args unknown_args"))
+    # Filter out the option terminator if present.
```

Comment:
Can you add an example of when this will happen?

The comment explains what's happening (which is clear from the code), but we should explain why this is needed.

Makes sense?